hathor/p2p/sync_v2/mempool.py

```diff
@@ -63,6 +63,8 @@ def run(self) -> Deferred[None]:
         self._is_running = True
         self.reactor.callLater(0, self._run)
 
+        # TODO Implement a stop() and call it after N minutes.
```

Comment:
TODO

Reply:
I'm planning to do it in a separate PR. I feel that this PR is already too big and this change is unrelated to the changes in this PR.